.github/workflows/publish_nuget.yml

```
@@ -1,7 +1,7 @@
 name: Publish Nuget
 on:
   push:
-    branches: [ main ]
+    branches: [main]
```

Revert accidental formatting changes

```suggestion
    branches: [ main ]
```